src/pages/DelegationPage/Index.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="delegation">
```

Comment:
since it is page a component, let's make it more semantic:  `<div>` --> `<main>`
unless everything is wrapped with `<main>` in app layout

Also if app follows classes naming convention based on the component name - I guess it should be `delegation-page`